include/ck_tile/ops/fmha/pipeline/tile_fmha_traits.hpp

```diff
@@ -33,6 +33,30 @@ struct TileFmhaTraits
     static constexpr index_t kBlockPerCu    = kBlockPerCu_;
 };
 
+template <bool kPadSeqLenQ_ /* padding for seqlen_q */,
```

Comment:
What's the the difference between TileFmhaTraits and TileFmhaBwdTraits?

Reply:
Removed  `TileFmhaBwdTraits`